src/OpenTelemetry.DependencyInjection/OpenTelemetry.DependencyInjection.csproj

```diff
@@ -0,0 +1,31 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <!-- OmniSharp/VS Code requires TargetFrameworks to be in descending order for IntelliSense and analysis. -->
+    <TargetFrameworks>netstandard2.0;net462</TargetFrameworks>
+    <Description>OpenTelemetry .NET dependency injection extensions</Description>
+    <RootNamespace>OpenTelemetry</RootNamespace>
+    <MinVerTagPrefix>core-</MinVerTagPrefix>
+    <Nullable>enable</Nullable>
```

Comment:
nit: The could be removed or updated accordingly now that these PRs are merged: https://github.com/open-telemetry/opentelemetry-dotnet/pull/3959, https://github.com/open-telemetry/opentelemetry-dotnet/pull/3960 , https://github.com/open-telemetry/opentelemetry-dotnet/pull/3964

Reply:
Updated